includes/class-wc-stripe-webhook-handler.php

```
@@ -1428,6 +1467,133 @@ public function process_deferred_webhook( $webhook_type, $additional_data, $noti
+	 * Used to decide whether an asynchronously-captured Stripe charge is orphaned (the order was
+	 * settled via a different payment method).
+	 *
+	 * @param WC_Order $order
+	 */
+	protected function is_order_paid_via_non_stripe_gateway( $order ): bool {
+		$payment_method = (string) $order->get_payment_method();
+		if ( '' === $payment_method ) {
+			return false;
+		}
+		return 'stripe' !== $payment_method && 0 !== strpos( $payment_method, 'stripe_' );
+	}
```

PHPMD (2.15.0)

[error] The method is_order_paid_via_non_stripe_gateway is not named in camelCase. (undefined)

(CamelCaseMethodName)